app/uk/gov/hmrc/helptosavefrontend/controllers/EligibilityCheckController.scala

```diff
@@ -194,9 +187,9 @@ class EligibilityCheckController @Inject()(val messagesApi: MessagesApi,
     import uk.gov.hmrc.helptosavefrontend.util.Toggles._
 
     userInfo match {
-      case None => Right(None)
-      case Some(ui) =>
-        FEATURE[Either[String, Option[NSIUserInfo]]]("outgoing-json-validation", app.configuration, Right(userInfo)) enabled() thenDo {
+      case None ⇒ Right(None)
+      case Some(ui) ⇒
+        FEATURE[Either[String, Option[NSIUserInfo]]]("outgoing-json-validation", app.configuration, Right(userInfo)) enabled () thenDo {
```

Comment:
why space after `enabled` ?

Reply:
i think it's because the line is using infix notation so to be consistent it's put the space after `enabled`